test/ising.jl

```diff
@@ -3,6 +3,34 @@ using LightGraphs
 using GraphPlot
 using CSV
 
+function _energy(config::Dict, couplings::Dict, cedges::Dict, n::Int)
+    eng = zeros(1,n)
+    for (i, j) ∈ keys(cedges)
```

Comment:
Please put this monstrosity in a single `for`

Reply:
This test (and many others) will be rewritten by @dexter2206. Before that happen, I would leave it as is.